File: dayOne/java/levelFour/TwoSum.java
public class TwoSum {

    public static int[] getTwoSum(int[] numbers, int target) {
        int[] newNumbers = new int[2];

        for (int index = 0; index < numbers.length; index++) {

            for (int innerIndex = index + 1; innerIndex < numbers.length; innerIndex++) {

                if (numbers[index] + numbers[innerIndex] == target) {
                    newNumbers[0] = index;
                    newNumbers[1] = innerIndex;

                    return newNumbers;
                }
            }
        }

        return newNumbers;
    }
}

File: dayOne/python/levelFour/two_sum.py
def get_two_sum(numbers, target):
    new_numbers = []

    for index in range(len(numbers)):

        for inner_index in range(len(numbers) - 1, index, -1):

            if numbers[index] + numbers[inner_index] == target:

                new_numbers.append(index)

                new_numbers.append(inner_index)

                return new_numbers

    return new_numbers